client/src/components/Common/BookingForm/BookingForm.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react'
+import Calender from '../../../Images/calendar_icon.png'
+import Clock from '../../../Images/clock_icon.png'
+import RoomIcon from '../../../Images/room_icon.png'
+import './bookingForm.css'
+import axios from 'axios'
+import { Link } from 'react-router-dom'
+import Popup from './Popup'
+class BookingForm extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      userId : 3,
+      date: '2020-05-23T11:00:00.000Z',
+      roomId:  5,
+      room_name : 'Moscow' ,
+      toggelReminder: false ,
+      showPopup : false
+    }
+  }
+
+  handelChange = (e) => {
+    const name = e.target.name
+    const value =e.target.value
+    this.setState({[name]: value})
+  }
+  handelChecked = ()=> {
+      this.setState({toggelReminder : !this.state.toggelReminder })
```

Comment:
sorry if I was unclear, I meant the name of the function to be called `toggleReminder`, the variable in the state should be just `reminder`. Or `isReminder`, which is a common good practice to name boolean variables to indicate that they are boolean